.env

```diff
@@ -0,0 +1,3 @@
+LOCAL_AGENT_PACKAGE=
+LOCAL_CONTROLLER_PACKAGE=
+LOCAL_CONNECTOR_PACKAGE=
```

Comment:
This file is to prevent the warning: `Variable not set, defaulting to blank string` from docker-compose when you run the start script without the env set